### ISSUE TITLE
Embeddings comparison blog: request for feedback

### ISSUE THREAD

**Issue by parulsethi** [RaRe-Technologies/gensim#1068]
@gojomo @akutuzov, I am writing this [blog](https://docs.google.com/document/d/1sVLCRlf1_Msoi7sd6byRmB5EYxOz9cEkCgdD49-Qw68/edit?usp=sharing) for the RaRe Technologies Incubator programme comparing wordrank, word2vec and fasttext. This is the corresponding ipynb 
https://github.com/parulsethi/gensim/blob/wordrank_wrapper/docs/notebooks/Wordrank_comparisons.ipynb. It would be great to have your opinions regarding the blog. Please comment in the doc link for suggestions.

**Comment by tmylk** [comment]
CC @anmol01gulati

**Comment by tmylk** [comment]
@parulsethi Thanks, the blog looks good. Let's publish it on the RaRe blog this week.

**Comment by jayantj** [comment]
@parulsethi  Looks good to me. Left some comments in the Google doc.

**Comment by parulsethi** [comment]
@jayantj thanks for the comprehensive review 
Can you please also tell what was the `no. of passes` over corpus for fasttext and if it uses multi-core implementation?

**Comment by tmylk** [comment]
cc @piskvorky

**Comment by piskvorky** [comment]
Left my comments in the doc -- mostly minor formatting & grammar.

My overall impression is that while the content is good, the text is way too long. It feels too pedestrian, which makes it hard to follow or see what the real conclusions or takeaways are.

Can you leave only the salient points and cut out the rest, streamline the narrative? Put some more structure into it = explicit sections, with clear headline & motivation & conclusion?

**Comment by parulsethi** [comment]
@piskvorky thanks for the comments. And sure, I'll try to organize it as suggested.

**Comment by tmylk** [comment]
Published https://rare-technologies.com/wordrank-embedding-crowned-is-most-similar-to-king-not-word2vecs-canute/
